modules/openxr/doc_classes/OpenXRAPIExtension.xml

```diff
@@ -156,6 +163,13 @@
 				Set the object name of an OpenXR object, used for debug output. [param object_type] must be a valid OpenXR [code]XrObjectType[/code] enum and [param object_handle] must be a valid OpenXR object handle.
 			</description>
 		</method>
+		<method name="set_render_region">
+			<return type="void" />
+			<param index="0" name="render_region" type="Rect2i" />
+			<description>
+				Sets the render region, overriding the normal render target rect dimensions.
```

Comment:
Unsure on this since I cannot speak about the actual implementation.
```suggestion
				Sets the render region to [param render_region], overriding the normal render target's rect.
```